src/nuxt/nuxt-wizard.ts

```diff
@@ -85,4 +84,16 @@ export async function runNuxtWizardWithTelemetry(
   });
 
   await addDotEnvSentryBuildPluginFile(authToken);
+
+  const nuxtConfig = await traceStep('load-nuxt-config', getNuxtConfig);
+
+  await traceStep('configure-sdk', async () => {
+    await addSDKModule(nuxtConfig, {
+      org: selectedProject.organization.slug,
+      project: selectedProject.slug,
+      url: selfHosted ? sentryUrl : undefined,
+    });
+
+    await createConfigFiles(selectedProject.keys[0].dsn.public);
+  });
```

Comment:
q: I handle error handling within the respective helpers. Would it be better to throw and wrap `traceStep` in a try catch?